cloudimg/ms_azure.py

```diff
@@ -354,6 +363,41 @@ def upload_callback(self, response):
 
         return self._upload_progress
 
+    def get_blob_copy_status(self, blob_client):
```

Comment:
If this method is intended for internal use only and not a part of the public cloudimg API, can you please name it with a leading `_` ?

And I do suggest you make it for internal use only, there doesn't seem to be a reason for users of `cloudimg` to call this method.

Reply:
done